ewms_pilot/pilot.py

```diff
@@ -207,19 +217,13 @@ async def process_msg_task(
     # call & check outputs
     LOGGER.info(f"Executing: {shlex.split(cmd)}")
     try:
-
         # await to start & prep coroutines
-        if debug_subdir:
-            with open(debug_subdir / "stdoutfile", "wb") as stdoutf, open(
-                debug_subdir / "stderrfile", "wb"
-            ) as stderrf:
-                proc = await asyncio.create_subprocess_shell(
-                    cmd,
-                    stdout=stdoutf,
-                    stderr=stderrf,
-                )
-        else:
-            proc = await asyncio.create_subprocess_shell(cmd)
+        with open(stdoutfile, "wb") as stdoutf, open(stderrfile, "wb") as stderrf:
+            proc = await asyncio.create_subprocess_shell(
+                cmd,
+                stdout=stdoutf,
+                stderr=stderrf,
+            )
```

Comment:
I'm curious whether the context manager needs to also include the "await to finish" block below this.

Reply:
I'm not sure but that would make sense. I'll do that anyways--can't hurt